internal/commands/test.go

```diff
@@ -196,6 +196,27 @@ func NewTestCommand(ctx context.Context) *cobra.Command {
 				}
 			} else {
 				namespaces = viper.GetStringSlice("namespace")
+				if len(namespaces) == 1 {
+					regexp, err := regexp.Compile(namespaces[0])
+					if err == nil {
+						var allNamespaces []string
+						allNamespaces, err = policy.GetNamespaces(regoFiles, compiler)
```

Comment:
as a plain string is a valid regex, I call GetNamespaces every time someone sets the --namespaces flag - not sure if this has a performance impact having to load all packages...